src/pages/PhoneNumberPage.tsx

```diff
@@ -0,0 +1,24 @@
+import "../components/modal/modal.scss";
+import Button from "../components/Button";
+import PhoneNumber from "../components/PhoneNumber";
+import {useState} from "react";
+
+const PhoneNumberPage = () => {
+  const [error, setError] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('375');
+
+  return (
+    <div className='field'>
+      <div className='modal'>
+        <h2 className='title'>Enter your phone number</h2>
+        <PhoneNumber setError={setError}
+                     error={error}
+                     phoneNumber={phoneNumber}
+                     setPhoneNumber={setPhoneNumber}/>
+        <Button buttonType={'submit'}
+                buttonText={'Submit code'}/>
```

Comment:
don't use that brackets `{}` if it's text variable; just put string like `buttonType="submit"`